sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -194,6 +196,24 @@ abstract class OrcSuite extends QueryTest with TestHiveSingleton with BeforeAndA
       Utils.deleteRecursively(location)
     }
   }
+
+  test("SPARK-21839: Add SQL config for ORC compression") {
+    val conf = sqlContext.sessionState.conf
+    assert(new OrcOptions(Map.empty[String, String], conf).compressionCodec == "SNAPPY")
```

Comment:
Add a comment here to explain the test scenario. 
// to test the default of `spark.sql.orc.compression.codec` is snappy

Reply:
Sure